src/main/scala/org/globalforestwatch/summarystats/forest_change_diagnostic/ForestChangeDiagnosticSummary.scala

```diff
@@ -123,6 +123,20 @@ object ForestChangeDiagnosticSummary {
         val idnPresence = gfwProCoverage.getOrElse("Indonesia", false)
         val argPresence = gfwProCoverage.getOrElse("Argentina", false)
 
+        // Currently, only do the area intersection with the detailed WDPA categories
+        // if location is in Argentina. Similarly, only do area intersection with
+        // Landmark (indigenous territories) if in Argentina.
+        // With lazy tile loading, the WDPA and landmark tiles are only loaded if
+        // argPresence is true.
+        val detailedWdpa = if (argPresence)
```

Comment:
Awesome if this works! Were you able to verify via logs it only loaded the data if this call was made?

Reply:
Yes,  I verified that it works!  I have those extra print statements in Layer.scala (which I will remove before checkin), and those print statements confirm that I don't load the detailedWdpa tiles for a particular location if the location is not Argentina (i.e. argPresence is false).  Same for the landmark tiles.